lib/stripe_model_callbacks/models/stripe_subscription.rb

```diff
@@ -100,4 +101,11 @@ def find_item_by_stripe_item(item)
       end
     end
   end
+
+  def latest_invoice_id(object)
+    return unless object.respond_to?(:latest_invoice)
+    return object.latest_invoice if object.latest_invoice.is_a?(String)
```

Comment:
What else can it be? Isn't it a question about if it is present?
```suggestion
    return object.latest_invoice if object.latest_invoice.present?
```

Reply:
It could be stripe_id  or stripe object. That is why I am checking if it is a String class.

Reply:
Is there a spec that reflects these different behaviours?